### PR TITLE
H2 Console Vulnerability

### PR DESCRIPTION
**Pull request by OndroMih**
Same as https://github.com/payara/Payara-Community-Documentation/pull/285 in Community.

### CODE REVIEW

File: docs/modules/ROOT/pages/security/security-fix-list.adoc [hunk 1]
```diff
@@ -9,6 +9,8 @@ can or have impacted Payara Server across releases:
 |=======================================================================
 |ID |CVSS v3.0 Base Score |Status |Summary |Release |Observations
 
+|https://nvd.nist.gov/vuln/detail/CVE-2021-42392[CVE-2021-42392] | N/A | N/A | Unauthenticated RCE in H2 Database Console |  | Unrelated to Payara Server. H2 Database Console doesn't launch the H2 Database Console and doesn't make it available in any way.
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/security/security-fix-list.adoc:12]: Sentence doesn't seems to be correct

**H2 Database Console** doesn't launch the **H2 Database Console** ...

**Review by rdebusscher — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/security/security-fix-list.adoc (hunk 1 above)
Comment (OndroMih) [reply docs/modules/ROOT/pages/security/security-fix-list.adoc:12]: You're right, I'll fix it.
```suggestion
|https://nvd.nist.gov/vuln/detail/CVE-2021-42392[CVE-2021-42392] | N/A | N/A | Unauthenticated RCE in H2 Database Console |  | Unrelated to Payara Platform. The Payara Platform doesn't launch the H2 Database Console and doesn't make it available in any way.
```

**Review by fturizo — APPROVED** [review]
Looks good, I'd change the "Unrelated to Payara Platform" to "Doesn't affect Payara Platform" so that the message is clearer.

File: docs/modules/ROOT/pages/security/security-fix-list.adoc [hunk 2]
```diff
@@ -9,6 +9,8 @@ can or have impacted Payara Server across releases:
 |=======================================================================
 |ID |CVSS v3.0 Base Score |Status |Summary |Release |Observations
 
+|https://nvd.nist.gov/vuln/detail/CVE-2021-42392[CVE-2021-42392] | N/A | N/A | Unauthenticated RCE in H2 Database Console |  | Unrelated to Payara Platform. The Payara Platform doesn't launch the H2 Database Console and doesn't make it available in any way.
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/security/security-fix-list.adoc:12]:
```suggestion
|https://nvd.nist.gov/vuln/detail/CVE-2021-42392[CVE-2021-42392] | N/A | N/A | Unauthenticated RCE in H2 Database Console |  | Doesn't affect Payara Platform. The Payara Platform doesn't launch the H2 Database Console and doesn't make it available in any way.
```

**Pull request merged**